EmVer/include/EmVerUtility.hpp

```diff
@@ -0,0 +1,5 @@
+#include "mbed.h"
+
+bool assertPin(PinName pinName, bool expected, int timeout);
+bool assertPinToggle(PinName pinName, int expectedOccurences, int timeout);
```

Comment:
I don't want to put a default timeout, since for every input we do have a specific timeout in mind due to the frequency.